lxd/container_delete.go

```diff
@@ -71,8 +30,14 @@ func containerDelete(d *Daemon, r *http.Request) Response {
 		return SmartError(err)
 	}
 
+	// TODO: i have added this not sure its a good idea (pcdummy)
```

Comment:
Doesn't seem like we really have a choice since removeContainer now takes a container struct instead of just the name. so not sure what's the alternative.